### PR TITLE
Fixes #11623: Create documentation on local node properties overide

### PR DESCRIPTION
**Pull request by ncharles**
https://www.rudder-project.org/redmine/issues/11623

### CODE REVIEW

File: 42_advanced_configuration_management/40_node_properties.txt [hunk 1]
```diff
@@ -4,6 +4,8 @@ Node properties can be found in the "properties" tab of each node in Rudder.
 
 Node properties can be modified using Rudder's API, see http://www.rudder-project.org/rudder-api-doc/#api-Nodes-updateNodeProperties
 
+Properties can also be defined on the node itself, to override locally properties
```
Comment (amousset) [inline 42_advanced_configuration_management/40_node_properties.txt]: missing . at the end

File: 42_advanced_configuration_management/40_node_properties.txt [hunk 2]
```diff
@@ -50,6 +52,9 @@ Those values are read from
 `/var/rudder/cfengine-community/inputs/properties/*.json`. All files are taken
 in order and override the previous ones - the last one wins.
 
+The agent seach for optionnal properties files `/var/rudder/local/properties.d/*.json`, and will define
```
Comment (amousset) [inline 42_advanced_configuration_management/40_node_properties.txt]: searches

File: 42_advanced_configuration_management/45_node_properties_in_directives.txt [hunk 3]
```diff
@@ -7,6 +7,10 @@ provide unique values for each node or be used in JavaScript expressions.
 Properties on nodes are defined using Rudder's REST API, with the 'Update Node properties' API call.
 More details in our http://www.rudder-project.org/rudder-api-doc[API documentation].
 
+Properties can also be defined directly on the nodes, by creating properties files 
+`/var/rudder/local/properties.d/*.json`/
```
Comment (amousset) [inline 42_advanced_configuration_management/45_node_properties_in_directives.txt]: misplaced backquote

**Comment by ncharles** [comment]
documentation states that we can have any namespace (and code also) to have node.somethingelse["foo"].
how should we include this information ?

**Comment by amousset** [comment]
Indeed, in this case we can give the example: "If you want to override a node property, you file should look like ..."

**Comment by ncharles** [comment]
Commit modified

File: 42_advanced_configuration_management/40_node_properties.txt [hunk 4]
```diff
@@ -39,6 +41,37 @@ In a mustache template, use:
 
 ----
 
+==== Local override
+
+The agent seaches for optionnal properties files `/var/rudder/local/properties.d/*.json`, and will override existing properties.
```
Comment (amousset) [inline 42_advanced_configuration_management/40_node_properties.txt]: searches

**Comment by ncharles** [comment]
Commit modified

File: 42_advanced_configuration_management/40_node_properties.txt [hunk 5]
```diff
@@ -50,6 +83,9 @@ Those values are read from
 `/var/rudder/cfengine-community/inputs/properties/*.json`. All files are taken
 in order and override the previous ones - the last one wins.
 
+The agent seaches for optionnal properties files `/var/rudder/local/properties.d/*.json`, and will define variables
```
Comment (amousset) [inline 42_advanced_configuration_management/40_node_properties.txt]: searches

**Pull request merged**